### ISSUE TITLE
Dynamic require of "util" is not supported

### ISSUE THREAD

**Issue by 7flash** [MoralisWeb3/react-moralis#96]
### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/react-moralis/blob/main/SECURITY.md).
- [x] I have searched through [existing issues](https://github.com/MoralisWeb3/react-moralis/issues?q=is%3Aissue) and the [Moralis Forum](https://forum.moralis.io/).
- [x] I can reproduce the issue with the [latest react-moralis release](https://github.com/MoralisWeb3/react-moralis/releases), [latest SDK release](https://github.com/MoralisWeb3/Moralis-JS-SDK/releases), and my server is updated to the latest version.

### Issue Description

When trying to import MoralisProvider into remix(React framework) giving this error in console:

```
Uncaught Error: Dynamic require of "util" is not supported
    at chunk-ZBXWLNC7.js:13
    at node_modules/web3-core-requestmanager/lib/index.js

```
### Steps + code to reproduce

1. in terminal
npx create-remix@latest

2. in terminal
npm install moralis react-moralis

3. in entry.client.jsx
import { MoralisProvider } from "react-moralis";

<img width="621" alt="Screenshot 2021-11-26 at 12 49 33" src="https://user-images.githubusercontent.com/4569866/143528478-79f43359-63c7-4e6c-a486-b43757ed5001.png">

**Comment by favealo** [comment]
👾

**Comment by ErnoW** [comment]
It seems to me that there is an issue in the build step. Could it be related to https://github.com/remix-run/remix/issues/348 ? 
I am not experienced with the remix framework though. But it could be an incompatibility web3.js and esbuild (used by remix).
The referenced line is
```
const { callbackify } = require('util');
```
It might not handle this import correctly. This should be fixed by web3.js or remix. There is nothing on our side that we can do (other than moving away from the web3.js dependency, which is in progress)

**Comment by ErnoW** [comment]
It should be resolved since v1.0, now that we do not depend on web3.js anymore.
